.github/workflows/build.yml

```
@@ -1,166 +1,45 @@
-          echo "- \`CraftUtils-${{ steps.artifact.outputs.version }}-shadow.jar\` - Shaded JAR with dependencies" >> $GITHUB_STEP_SUMMARY
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo "### 📊 Build Information:" >> $GITHUB_STEP_SUMMARY
-          echo "- **Java Version:** 21 (Temurin)" >> $GITHUB_STEP_SUMMARY
-          echo "- **Gradle Version:** $(./gradlew --version | grep Gradle | head -1)" >> $GITHUB_STEP_SUMMARY
-          echo "- **Target Minecraft:** 1.21.8+" >> $GITHUB_STEP_SUMMARY
-          echo "- **Server Software:** Paper/Spigot" >> $GITHUB_STEP_SUMMARY
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Validate Gradle wrapper
+      uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.7)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)